src/main/java/pl/pawit/Main.java

```diff
@@ -1,7 +1,37 @@
 package pl.pawit;
 
+import java.io.IOException;
+import java.nio.charset.Charset;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Scanner;
+
 public class Main {
-    public static void main(String[] args) {
-        System.out.println("Hello world!");
+    public static void main(String[] args) throws IOException {
+
+
+        Scanner scanner = new Scanner(System.in);
+
+        boolean continueGame = true;
+        while (continueGame) {
+            System.out.println("1.Start the game");
+            System.out.println("2.Skorzystaj z kola ratunkowego");
+            System.out.println("3.End the game");
+            continueGame = false;
+
+            int userChoice = scanner.nextInt();
+            ;
+            switch (userChoice) {
```

Comment:
moze poki co zrezygnujemy ze switcha. I tutaj na ten moment po prostu wlacza sie aplikacja i odpala sie pierwsze pytanie, uzytkownik odpowiada poprawnie to wlacza sie kolejne i tyle. Switche dorobimy pozniej jak juz zczytywanie pytan i odpowiedzi bedzie dzialalo poprawnie